docs/examples/content-type/runtime.py

```diff
@@ -21,16 +21,16 @@
 
 
 @register_input_codec
-class PillowCodec(NumpyCodec):
+class PillowCodec(InputCodec):
     ContentType = "img"
     DefaultMode = "L"
 
     @classmethod
-    def can_encode(cls, payload: Image) -> bool:
-        return isinstance(payload, Image)
+    def can_encode(cls, payload: Image.Image) -> bool:
```

Comment:
Sure! It's because `Image` is a module and not a class [1] and thus not a valid type.

[1]
```
>>> from PIL import Image
>>> Image
<module 'PIL.Image' from '/python/3.10.12/lib/python3.10/site-packages/PIL/Image.py'>
>>> Image.Image
<class 'PIL.Image.Image'>
```